easybuild/framework/easyblock.py

```diff
@@ -3130,6 +3130,11 @@ def _sanity_check_step_multi_deps(self, *args, **kwargs):
     def sanity_check_rpath(self, rpath_dirs=None, check_readelf_rpath=True):
         """Sanity check binaries/libraries w.r.t. RPATH linking."""
 
+        if self.cfg.get('check_readelf_rpath') is False:
+            self.log.info(
+                "RPATH checking is disabled due to the easyconfig setting 'check_readelf_rpath=False' ... skipping"
+            )
+            return []
```

Comment:
This return is way too early, we only want to skip the `readelf` part below, so just rely on the existing `check_readelf_rpath` conditional guard that's already in place?